mobile/lib/ui/sharing/album_participants_page.dart

```diff
@@ -54,7 +54,7 @@ class _AlbumParticipantsPageState extends State<AlbumParticipantsPage> {
   Future<void> _navigateToAddUser(bool addingViewer) async {
     await routeToPage(
       context,
-      AddParticipantPage(widget.collection, addingViewer),
+      AddParticipantPage(widget.collection as List<Collection>, addingViewer),
```

Comment:
This will throw an error because it's trying to cast an `int` to a `List<int>`
Just pass `[widget.collection]` instead.